src/Yasumi/Holiday.php

```diff
@@ -56,19 +56,19 @@ class Holiday extends DateTime implements JsonSerializable
     public const DEFAULT_LOCALE = 'en_US';
 
     /**
-     * Pseudo-locale representing the short name (internal name) of the holiday.
+     * Pseudo-locale representing the holiday key.
      */
-    public const LOCALE_SHORT_NAME = 'shortName';
+    public const LOCALE_KEY = '_key';
```

Comment:
The problem is that locales may use a two-letter or three-letter language code. The string “key” might conflict with a real language code (the code is not currently assigned).

So the underscore is an attempt to prevent collisions. Another suggestion is e.g. `yasumi_key` or `zz_key`. I think they are all equally bad, so suggestions are welcome :-)

Reply:
Indeed. Let's leave it with the underscore.